src/PaymentGateways/PayPalCommerce/onBoardingRedirectHandler.php

```diff
@@ -339,16 +375,22 @@ private function registerPayPalSSLNotice() {
 		if ( ! is_ssl() ) {
 			Give_Admin_Settings::add_error(
 				'paypal-ssl-error',
-				'There was a problem registering your site\'s webhook with PayPal. In order for to register
+				esc_html__(
```

Comment:
Can unwrap text?

Reply:
Per our conversation on Slack, I'm going to keep the wrapping: https://givewp.slack.com/archives/CM9NDF2QY/p1596818230008900?thread_ts=1596816407.002500&cid=CM9NDF2QY